src/main/java/com/plivo/api/models/message/MmsMedia.java

```diff
@@ -0,0 +1,38 @@
+package com.plivo.api.models.message;
+
+import com.plivo.api.models.base.BaseResource;
+
+import java.util.List;
+
+public class MmsMedia extends BaseResource {
+  private String content_type;
+  private String media_id;
+  private String media_url;
+  private String message_uuid;
+  private Long size;
```

Comment:
Add comment. size in what bytes? Also any reason for not using primitive (long vs Long)